src/components/FileUploader/FileUploader.tsx

```diff
@@ -55,7 +55,7 @@ const FileUploader = (props: PropTypes) => {
       )) || <ClipLoader />}
       {file && <div>{file.name}</div>}
       <div className="file-upload__footer">
-        <button className="file-upload__button" onClick={onSubmit}>
+        <button className="button button__primary" onClick={onSubmit}>
```

Comment:
<img width="1308" alt="image" src="https://github.com/nimblehq/ic-nextjs-prabesh-dahal/assets/696529/028d4678-0512-46ad-a550-7656fdcbec2f">

That is not consistent with the file above:

```suggestion
        <button className="button button--primary" onClick={onSubmit}>
```